src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/View/shipping.html.twig

```diff
@@ -86,14 +86,6 @@
   {% if orderForViewing.shipping.carrierModuleInfo %}
     {{ orderForViewing.shipping.carrierModuleInfo|raw }}
   {% endif %}
-
-  {% if orderForViewing.shipping.recycledPackaging %}
-    <span class="badge badge-success">{{ 'Recycled packaging'|trans({}, 'Admin.Orderscustomers.Feature') }}</span>
-  {% endif %}
-
-  {% if orderForViewing.shipping.giftWrapping %}
-    <span class="badge badge-success">{{ 'Gift wrapping'|trans({}, 'Admin.Orderscustomers.Feature') }}</span>
-  {% endif %}
 {% else %}
   <p class="text-center mb-0">
     {{ 'Shipping does not apply to virtual orders'|trans({}, 'Admin.Orderscustomers.Feature') }}
```

Comment:
I could not find this wording in the back office, would it be possible to provide a screenshot please? Thank you !

Reply:
Order page when gift wrapping and recycled packaging is enabled, see the issue the complete step is described

![image](https://user-images.githubusercontent.com/14963751/97311297-801b4880-1864-11eb-952a-677b63c0eadc.png)

Reply:
Thank you but I was just missing the wording "Shipping does not apply to virtual orders". 😊 I couldn't find it

Reply:
You need to create a virtual order probably, anyway it's not related to this PR at all

Reply:
Sorry, I haven't been using GitHub for a long time and I didn't know that wordings written on a white background were not affected by the PR. Anyway, the wording is approved now. 😄